wagtail/core/blocks/list_block.py

```diff
@@ -79,6 +89,7 @@ def value_omitted_from_data(self, data, files, prefix):
     def clean(self, value):
         result = []
         errors = []
+        non_block_errors = ErrorList()
```

Comment:
``non_block_errors`` doesn't seem to ever be set in this method

Reply:
Yep, this is a bit of a placeholder right now until we have the min_num / max_num params to make use of it.